scripts/constants.py

```diff
@@ -15,10 +15,10 @@
 import sys
 
 # Kubernetes branch to get the OpenAPI spec from.
-KUBERNETES_BRANCH = "release-1.15"
+KUBERNETES_BRANCH = "release-1.16"
 
 # client version for packaging and releasing.
-CLIENT_VERSION = "11.0.0-snapshot"
+CLIENT_VERSION = "12.0.0a1"
```

Comment:
This PR targets the master branch. I think we should first generates a `12.0.0-snapshot` client in the master branch with API changes from upstream `release-1.16`. Then generates a `12.0.0a1` client in the `release-12.0` branch with only version constant changes.